bin/add_frontend_server.rb

```diff
@@ -42,6 +46,10 @@ def execute_ssh_on_channel(channel, command, exit_error_string)
   opts.on('-h', '--help', 'Print this') { puts options; exit }
 end.parse!
 
+unless ['production', 'adhoc'].include?(options['environment'])
+  throw 'Environments other than production are currently unsupported'
```

Comment:
My mistake - I didn't want to create in prod while testing this. I'll take adhoc out.